internal/pkg/template/env.go

```diff
@@ -109,8 +109,8 @@ type EnvOpts struct {
 	ArtifactBucketKeyARN string
 
 	VPCConfig         VPCConfig
-	PublicHTTPConfig  HTTPConfig
-	PrivateHTTPConfig HTTPConfig
+	PublicHTTPConfig  PublicHTTPConfig
+	PrivateHTTPConfig PrivateHTTPConfig
```

Comment:
Awesome, ty so much for also refactoring `PrivateHTTPConfig`! <img src="https://emojis.slackmojis.com/emojis/images/1643515051/10804/meow_love.png?1643515051" width=15px />